js/controls/speciesmap_controls.js

```diff
@@ -443,9 +540,24 @@ var control_speciesmap_addcontrols;
       finishButtonId: 'speciesmap_finishbutton_control',
       messageId: 'speciesmap_controls_messages',
       messageClasses: '',
-      featureLabel: 'Grid: ${sRef}\nSpecies: ${count}',
+      //featureLabel: 'Grid: ${sRef}\nSpecies: ${count}',
```

Comment:
No need to leave commented out code in place?
Otherise the pull request looks good, thanks.

Reply:
Sorry - overlooked it. I will remove.